pkg/ddl/backfilling.go

```diff
@@ -823,8 +823,12 @@ func (dc *ddlCtx) writePhysicalTableRecord(
 	reorgInfo *reorgInfo,
 ) error {
 	startKey, endKey := reorgInfo.StartKey, reorgInfo.EndKey
-
 	if err := dc.isReorgRunnable(reorgInfo.Job.ID, false); err != nil {
+		if errors.ErrorEqual(err, dbterror.ErrNotOwner) {
```

Comment:
Is it possible for us add one UT?

Reply:
No, we cannot simulate owner change two times in UT.